gradle-plugin/src/main/kotlin/io/spine/protodata/gradle/Plugin.kt

```diff
@@ -75,160 +72,48 @@ public class Plugin : GradlePlugin<Project> {
         val extension = Extension(target)
         target.extensions.add("protoData", extension)
         val protoDataConfiguration = target.configurations.create("protoData")
-        createLaunchTask(target, extension, protoDataConfiguration)
+        target.sourceSets.forEach { sourceSet ->
+            val launch = createLaunchTask(target, extension, protoDataConfiguration, sourceSet)
+            target.javaCompileFor(sourceSet)?.dependsOn(launch)
+        }
 
         val resource = Plugin::class.java.classLoader.getResource("version.txt")!!
```

Comment:
Please have `version.txt` defined as a constant. It's used at least twice in the code.